modules/order/src/AdjustableInterface.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Drupal\commerce_order;
+
+/**
+ * Defines an interface for objects that interact with Adjustments.
+ *
+ * @todo Provide a trait? But that'd assume the field is always 'adjustments'
```

Comment:
Yeah, the need for calculateTotals on the order makes me thing we should not be abstracting this. At least not yet.